data_products/README.md

```diff
@@ -13,9 +13,8 @@ The simulations were all performed in MEGAlib, with an accurate mass model of th
 
 We have included many combinations of the source simulations and background to allow for flexibility and further testing with these files. All files are either the .npz zipped numpy array format for the CDS-binned response matrix or background model, or the MEGAlib photon list .tra.gz format. MEGAlib was used to perform all of these simulations, and the source models are described in detail below.
 
-Within this directory, there are 2 background models and 3 response matrices. 
+Within this directory, there is 1 background model and 3 response matrices. 
 - **Scaled_Ling_BG_1x.npz**: background model generated from C. Karwin's scaled 1x Ling background simulation 
-- **Scaled_Ling_BG_3x.npz**: background model generated from C. Karwin's scaled 3x Ling background simulation for more statistics
```

Comment:
As far as I can tell, the Scaled_Ling_BG_3x.npz file is still in the data_products folder. This should be removed.